docs/fides/docs/installation/configuration.md

```diff
@@ -10,16 +10,27 @@ After initializing fidesctl, a default configuration file will be generated and
 
 ```toml title="fidesctl.toml"
 [api]
+
+# If FIDESCTL_TEST_MODE env var is set to True, the test_database_url
+# will be used instead of the database URL to avoid overwriting production data
+test_database_url = "postgres:fidesctl@fidesctl-db:5432/fidesctl_test"
+
+# The SQLAlchemy connection string used to connect to the database
+# https://docs.sqlalchemy.org/en/14/core/engines.html#postgresql
```

Comment:
I don't believe anything about the connection string is specific to SQLAlchemy, and naming SQLAlchemy here + linking to the docs leaks implementation details.
```suggestion
# The PostgreSQL database connection string for the fidesctl database
```

Reply:
regardless of the tool, sqlalchemy has a very useful page on connection strings. what is the negative of leaking implementation details?

Reply:
In general it's a bad practice. It opens up potential attack vectors, couples the documentation to the implementation, and may confuse inexperienced users who could be led to believe that these are somehow SQLAlchemy-specific. I don't believe we need to guide users on building connection strings as part of our docs (similar to [my below comment](https://github.com/ethyca/fides/pull/348#discussion_r801037507), I think we should decompose the config into it the connection's components and build the string ourselves), but if you'd like to link to something, why not [the pSQL docs](https://www.postgresql.org/docs/14/libpq-connect.html#LIBPQ-CONNSTRING) instead?

Reply:
added